models/Counsel.js

```diff
@@ -0,0 +1,51 @@
+const mongoose = require("mongoose");
+
+const counselSchema = new mongoose.Schema({
+  counselee: {
+    type: String,
+    required: [true, "카운슬리는 반드시 필요합니다."],
+  },
+  counselor: {
+    type: String,
+    index: true,
+  },
```

Comment:
```suggestion
  counselee: {
      type: mongoose.Schema.Types.ObjectId,
      ref: "User",
   },
  counselor: {
     type: mongoose.Schema.Types.ObjectId,
     ref: "User",
   },
```
counslee, counselor 둘 다 User 를 참조해야할 것 같아요~!

Reply:
앗 그렇네요